packages/connector-aliyun-dm/jest.config.js

```diff
@@ -0,0 +1,17 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+  setupFilesAfterEnv: ['jest-matcher-specific-error'],
+  globals: {
+    'ts-jest': {
+      tsconfig: 'tsconfig.test.json',
+    },
+  },
+  moduleNameMapper: {
+    '^@/(.*)$': '<rootDir>/src/$1',
+    '^jose/(.*)$': '<rootDir>/node_modules/jose/dist/node/cjs/$1',
+  },
+  coveragePathIgnorePatterns: ['/node_modules/', '/build/'],
+  coverageReporters: ['text-summary', 'lcov'],
+};
```

Comment:
Better write in `.ts`.
You can do it later.

Reference: `packages/ui/jest.config.ts`